redbot/core/events.py

```diff
@@ -99,10 +100,65 @@ async def on_ready():
+                requires_python = data["info"]["requires_python"]
+                current_python = platform.python_version()
+                extra_update = _(
+                    "\n\nWhile the following should work in most scenarios as it is "
+                    "based on your current OS, environment, and Python version, "
+                    "**we highly recommend you to read the update docs at <{docs}> and "
+                    "make sure there is nothing else that "
+                    "needs to be done during the update.**"
+                ).format(docs="https://docs.discord.red/en/stable/update_red.html",)
+                if expected_version(current_python, requires_python):
+                    installed_extras = []
+                    for extra, reqs in red_pkg._dep_map:
+                        if extra is None:
+                            continue
+                        try:
+                            pkg_resources.require(req.name for req in reqs)
+                        except pkg_resources.DistributionNotFound:
+                            pass
+                        else:
+                            installed_extras.append(extra)
+
+                    if installed_extras:
+                        package_extras = f"[{','.join(installed_extras)}]"
+                    else:
+                        package_extras = ""
+
+                    extra_update += _(
+                        "\n\nTo update your bot, first shutdown your "
+                        "bot then open a window of {console} (Not as admin) and "
+                        "run the following:\n\n"
+                    ).format(
+                        console=_("Command Prompt")
+                        if platform.system() == "Windows"
+                        else _("Terminal")
+                    )
+                    extra_update += '```"{python}" -m pip install -U Red-DiscordBot{package_extras}```'.format(
+                        python=sys.executable, package_extras=package_extras
+                    )
+
+                else:
+                    extra_update += _(
+                        "\n\nYou have Python `{py_version}` and this update "
+                        "requires `{req_py}`; you cannot simply run the update command.\n\n"
+                        "You will need to follow the update instructions in our docs above, "
+                        "if you still need help updating after following the docs go to our "
+                        "#support channel in <https://discord.gg/red>"
+                    ).format(
+                        py_version=current_python,
+                        req_py=requires_python,
+                        docs="https://docs.discord.red/en/stable/install_windows.html"
+                        if platform.system() == "Windows"
+                        else "https://docs.discord.red/en/stable/install_linux_mac.html",
+                    )
```

Comment:
`{docs}` isn't in the string anymore so this is unneeded
```suggestion
                    ).format(py_version=current_python, req_py=requires_python)
```